modules/etcd-tuning-parameters.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_content-type: PROCEDURE
+[id="etcd-tuning-parameters_{context}"]
+= Setting tuning parameters for etcd
+
+You can set the control plane hardware speed to `""` (the default), `"Standard"`, or `"Slower"`.
+
+The default setting allows the system to decide which speed to use. This value enables upgrades from versions where this feature does not exist, as the system can select values from previous versions.
+
+By selecting one of the other values, you are overriding the default. If you see many leader elections due to timeouts or missed heartbeats and your system is set to `""` or `"Standard"`, set the hardware speed to `"Slower"` to make the system more tolerant to the increased latency.
+
+:FeatureName: Timer selectable profiles for etcd
```

Comment:
@dusk125 I'm not exactly sure what the feature name is, so feel free to weigh in if this is wrong.

Reply:
The FeatureName from the Jira is "etcd timers selectable profiles", but maybe something along the lines of "Tuning etcd latency tolerances" or similar would be better?